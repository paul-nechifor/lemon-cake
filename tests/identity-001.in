(identity 1234)
(identity :asdf)
(identity (~ (x) (+ 1 x)))
