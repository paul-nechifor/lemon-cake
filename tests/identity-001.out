1234
asdf
(~ (x) (+ 1 x))
